src/v/utils/vint.h

```diff
@@ -12,36 +12,57 @@
 #pragma once
 #include "bytes/bytes.h"
```

Comment:
Should that change be a part of other commit ?

Reply:
Sorry didn't see this comment. I placed it here as this is where the introduction of the stream decoder method is necessitated